src/mlpack/tests/load_save_test.cpp

```diff
@@ -49,7 +49,7 @@ TEST_CASE("NotExistLoad", "[LoadSaveTest]")
 /**
  * Make sure load fails if the file extension is wrong in automatic detection mode.
  */
-TEST_CASE("WrongExtensionWrongLoad", "[LoadSaveTest]")
+TEST_CASE("WrongExtensionWrongLoad", "[LoadSaveTest][tiny]")
```

Comment:
I understand that running these tests is beneficial, but please keep in mind that on a device like the VoCore, you have lower chances of loading the dataset using the loader. In most cases, you will be capturing data from a sensor or a similar source.